components/Co-workingSpace/ModalComp.tsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react'
+import Button from "@/components/atoms/Button/Button";
+import Dialog from '@mui/material/Dialog';
```

Comment:
Hey @daveeedu, great work here man.

I noticed that you introduced another package(MUI) to the project to complete the modal component.
At the moment we are choosing to stick to using headlessUi as our extra component library/framework.
They have an excellent modal component which you can find here [HeadlessUiModal](https://headlessui.com/react/dialog)    
Or better still you can use the modal component we have that is powered by headlessUi which you can find `@components/atoms/Modal/Modal.tsx`

Reply:
![dave](https://github.com/AfriSplash-Remotely/Afrisplash-Frontend/assets/46542941/2d65ed74-a807-43a2-ba49-48d989425fce)
Can you also look into the heading text here, it isn't properly aligned
Thank you

Reply:
Hello @sammy-code98, I have made the changes you requested. Thank you for the feedback.